src/psyclone/tests/test_files/dynamo0p3/15.1.1_X_plus_Y_builtin.f90

```diff
@@ -0,0 +1,50 @@
+! -----------------------------------------------------------------------------
+! BSD 3-Clause License
+!
+! Copyright (c) 2017, Science and Technology Facilities Council
+! However, it has been created with the help of the GungHo Consortium,
```

Comment:
Please remove lines 5 and 6.

Reply:
Corrected the BSD license, as well as in all other test files.